xla/service/gpu/transforms/cudnn_fused_mha_rewriter_test.cc

```diff
@@ -160,17 +166,33 @@ ENTRY main.6 {
   convert.2 = bf16[16,16,256]{2,1,0} convert(reduce.23)
   broadcast.4 = bf16[16,16,256,256]{3,2,1,0} broadcast(convert.2), dimensions={0,1,2}
   divide = bf16[16,16,256,256]{3,2,1,0} divide(exponential.1, broadcast.4)
-  ROOT dot.1 = bf16[16,16,256,64]{3,2,1,0} dot(divide, Arg_2.3), lhs_batch_dims={0,1}, lhs_contracting_dims={3}, rhs_batch_dims={0,1}, rhs_contracting_dims={2}, metadata={}
+  ROOT dot.1 = bf16[16,16,256,HEAD_DIM]{3,2,1,0} dot(divide, Arg_2.3), lhs_batch_dims={0,1}, lhs_contracting_dims={3}, rhs_batch_dims={0,1}, rhs_contracting_dims={2}, metadata={}
 })";
 
+std::string ReplacePlaceholder(absl::string_view str,
```

Comment:
No need for a custom replace function. absl::StrReplaceAll should provide the functionality you need.